src/Avalonia.Visuals/Media/TextFormatting/TextLayout.cs

```diff
@@ -35,7 +35,7 @@ public class TextLayout
         /// <param name="maxLines">The maximum number of text lines.</param>
         /// <param name="textStyleOverrides">The text style overrides.</param>
         public TextLayout(
-            string text,
+            string? text,
             Typeface typeface,
             double fontSize,
             IBrush foreground,
```

Comment:
The `foreground` is also nullable. Passing null still lets you measure everything but nothing is drawn in the end.